tests/std/tests/P0898R3_concepts/test.cpp

```diff
@@ -1425,9 +1425,6 @@ namespace test_constructible_from {
     STATIC_ASSERT(test<int&, ExplicitTo<int&>>());
     STATIC_ASSERT(test<int const&, ExplicitTo<int&>>());
     STATIC_ASSERT(test<int const&, ExplicitTo<int&>&>());
-#if defined(__clang__) // TRANSITION, VSO-1357053 (MSVC) and VSO-1357056 (EDG)
-    STATIC_ASSERT(test<int const&, ExplicitTo<int&&>>());
-#endif // TRANSITION, VSO-1357053 (MSVC) and VSO-1357056 (EDG)
 

```

Comment:
There's a second occurrence: https://github.com/microsoft/STL/blob/bfa2cb2a6dc168653a2752cb91521e8d0b4a4e69/tests/std/tests/P0898R3_concepts/test.cpp#L1460-L1462

Reply:
Argh. Validated I've removed all occurrences of `\b135705[36]\b`.